src/CarbonAware.WebApi/test/integrationTests/CarbonAwareControllerTests.cs

```diff
@@ -20,6 +20,8 @@ public class CarbonAwareControllerTests : IntegrationTestingBase
     private string bestLocationsURI = "/emissions/bylocations/best";
     private string currentForecastURI = "/emissions/forecasts/current";
     private string batchForecastURI = "/emissions/forecasts/batch";
+    private string actualHistoricalURI = "/emissions/average-carbon-intensity";
```

Comment:
The term "actual" has mostly been used internally to differentiate, but it is not a technical term used in the codebase.  Prefer a variable name like `averageCarbonIntensityURI`